src/information-panel/information-panel.scss

```
@@ -187,3 +190,38 @@
     top: $iui-m * -0.5;
   }
 }
+
+/// Labels and inputs aligned using css grid
+@mixin iui-information-body-content {
+  .iui-input-label {
+    font-weight: normal;
+    @include themed {
+      color: t(iui-text-color-muted);
+    }
+  }
+
+  &:not(.iui-inline) {
+    // add bottom margin between pairs of label+input
+    > *:nth-child(even):not(:last-child) {
+      margin-bottom: 11px;
```

Variable should work here:
```suggestion
      margin-bottom: $iui-baseline;
```

---

Fixed already